### DISCUSSION TITLE
nut-input disabled 默认值问题

### DISCUSSION THREAD

**Discussion by hero-xiaoye** [jdf2e/nutui#3109] [Q&A] [answered]
引入nut-input后并未显示设置disalbed为true，但编译为飞书小程序后，所有使用该组件的位置都添加了disabled属性。排查并未发现手动设置了nut-input的disabled的默认值。应该如何重置？或可能设置该默认值的途径是什么？以便排查

**Comment by eiinu** [comment, marked as answer]
这是 vue3.4.28 引入的问题，升级版本至 3.4.30 就可以了。

**Reply by hero-xiaoye** [reply]
酷，问题已解决，**谢谢**
